ci_scripts/ci_post_clone.sh

```diff
@@ -16,10 +16,11 @@ install_dependencies() {
         brew install swiftlint
     elif [[ "$CI_WORKFLOW" == "cocoapods"* ]]; then
         install_ruby
-    elif [[ "$CI_WORKFLOW" == "objectserver"* ]] || [[ "$target" == "swiftpm"* ]]; then
+    elif [[ "$CI_WORKFLOW" == "sync"* ]]; then
+    # elif [[ "$CI_WORKFLOW" == "sync"* ]] || [[ "$CI_WORKFLOW" == "swiftpm"* ]]; then
```

Comment:
Are we not running the sync tests in SPM?

Reply:
We're not running the sync tests *anywhere*, and trying and failing to install baas now results in the non-sync spm tests not running.